wordpress-mobile/WordPress-Android
# Fix large font size accessibility issue on migration screens by making the content scrollable

Fixes #17467

This PR fixes the accessibility issues on the WordPress to Jetpack Migration Flow UI namely:

- When the largest font size is set via Accessibility settings, on some devices part of the text might not be fitting on the screen.

It accomplishes this by making the content of the Notifications and Success screens scrollable.

> 🏁 **All these changes are behind a temporary feature flag.**

To test:

#### 1️⃣ Test with largest font size

0. Open the Jetpack app & login
1. Go to `Me` → `App Settings` → `Debug Settings`
2. Enable the `JetpackMigrationFlowFeatureConfig` & restart the app
3. Toggle largest font size via accessibility settings
   a. Open the device `Settings` app, go to `Accessibility` → `Display size and text`
   b. Set `Font size` to the max
4. Return to / Open the Jetpack app
5. Tap `Continue` on the Welcome Step
6. **Verify** that the Notifications screen can be scrolled if the text is not fitting
7. Tap `Continue` on the Notifications step
8. **Verify** that the Success screen can be scrolled if the text is not fitting

#### 2️⃣ Regression Testing - with default font size
1. Reset Accessibility font size settings
   a. Open the device `Settings` app, go to `Accessibility` → `Display size and text`
   b. Tap `Reset settings` at the bottom, or restore the initial font size
2. Open the Jetpack app
3. Tap `Continue` on the Welcome Step
4. **Verify** that Notifications screen matches the design (no regression)
5. Tap `Continue` on the Notifications step
6. **Verify** that the Success screen matches the design (no regression)

## Preview

_**Note** that I've also set `Display Size` to max via Accessibility Settings, otherwise the text would fit on one screen on the Pixel 6 Emulator._

https://user-images.githubusercontent.com/4588074/202516284-56efea91-1056-46fa-a087-5930d282f409.mp4

## Regression Notes

1. Potential unintended areas of impact
   Migration screens behind the feature flag.

6. What I did to test those areas of impact (or what existing automated tests I relied on)
   Manually tested.

7. What automated tests I added (or what prevented me from doing so)
   N/a.

PR submission checklist:
